.travis.yml

```diff
@@ -37,6 +37,8 @@ before_script:
 script:
   - grails refresh-dependencies --non-interactive
   - grails test-app --non-interactive --stacktrace
+  - chmod a+x test/runit/run_tests.R
```

Comment:
You don't need this line. Only if you wanted to run like this `./test/runit/run_tests.R`